lib/cache.js

```diff
@@ -37,7 +37,9 @@ Cache.prototype.del = function(key) {
 
 // Clear cache elements that start with prefix
 //   prefix - string, to compare with beginning of the key
-//   fn - function (optional), `(key, val)` to be called on each key
+//   fn - function, (key, val) (optional)
```

Comment:
Can we make this without comma? (i.e. `fn - function(key, val) (optional)`)

Reply:
This can be done, but for this purpose parser need to be changed to support explicitly this style of commenting functions arguments. One option is to change it as you suggested and leave it until parser will be ready to support such comments type.

Reply:
But you are already changing the parser in https://github.com/metarhia/metaschema/pull/107, so is there any problem of including this syntax there? The current parser cannot parse any of the styles here.